app/Resources/assets/scss/main.scss

```diff
@@ -144,6 +144,24 @@ body#homepage {
 // Page: 'login'
 // --------------------------------------------------
 body#login {
+    #login-users-help p {
```

Comment:
It's true ... but we could use it for example in the 403 error page, where the `id` of the body is `error`.